netci.groovy

```diff
@@ -209,67 +213,79 @@ def buildArchConfiguration = ['Debug': 'x86',
-        ['Debug', 'Release'].each { configurationGroup ->
+    ['netcoreapp', 'netfx'].each { targetGroup ->
+        (targetGroupOsMap[targetGroup]).each { osName ->
+            ['Debug', 'Release'].each { configurationGroup ->
 
-            def osForMachineAffinity = osName
-            if (osForMachineAffinity == 'PortableLinux') {
-                // Portable Linux builds happen on RHEL7.2
-                osForMachineAffinity = "RHEL7.2"
-            }
+                def osForMachineAffinity = osName
+                if (osForMachineAffinity == 'PortableLinux') {
+                    // Portable Linux builds happen on RHEL7.2
+                    osForMachineAffinity = "RHEL7.2"
+                }
 
-            def archGroup = "x64"
-            def newJobName = "outerloop_${osShortName[osName]}_${configurationGroup.toLowerCase()}"
+                def archGroup = "x64"
+                def osJobName = osShortName[osName]
 
-            def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
-                steps {
-                    if (osName == 'Windows 10' || osName == 'Windows 7' || osName == 'Windows_NT') {
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd -${configurationGroup}")
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build-tests.cmd -${configurationGroup} -outerloop -- /p:IsCIBuild=true")
-                    }
-                    else if (osName == 'OSX10.12') {
-                        shell("HOME=\$WORKSPACE/tempHome ./build.sh -${configurationGroup.toLowerCase()}")
-                        shell("HOME=\$WORKSPACE/tempHome ./build-tests.sh -${configurationGroup.toLowerCase()} -outerloop -- /p:IsCIBuild=true")
-                    }
-                    else if (osName == 'CentOS7.1') {
-                        // On Centos7.1, the cmake toolset is currently installed in /usr/local/bin (it was built manually).  When
-                        // running sudo, that will be typically eliminated from the PATH, so let's add it back in.
-                        shell("sudo PATH=\$PATH:/usr/local/bin HOME=\$WORKSPACE/tempHome ./build.sh -${configurationGroup.toLowerCase()}")
-                        shell("sudo PATH=\$PATH:/usr/local/bin HOME=\$WORKSPACE/tempHome ./build-tests.sh -${configurationGroup.toLowerCase()} -outerloop -- /p:IsCIBuild=true")
-                    }
-                    else {
-                        def portableLinux = (osName == 'PortableLinux') ? '-portableLinux' : ''
-                        shell("sudo HOME=\$WORKSPACE/tempHome ./build.sh -${configurationGroup.toLowerCase()} ${portableLinux}")
-                        shell("sudo HOME=\$WORKSPACE/tempHome ./build-tests.sh -${configurationGroup.toLowerCase()} -outerloop -- /p:IsCIBuild=true")
+                if (targetGroup == 'netfx') {
```

Comment:
Sorry I miss-spoke, as I see you are overriding osJobName with netfx. I think we should just go ahead and add the targetgroup to the job name for all cases.